src/manage/src/commands/bulk-reprocess-regions.ts

```diff
@@ -0,0 +1,98 @@
+import { Command, flags } from "@oclif/command";
+import { IArg } from "@oclif/parser/lib/args";
+import { existsSync, readFileSync, mkdirSync } from "fs";
+import UpdateRegion from "./update-region";
+import ProcessGeojson from "./process-geojson";
+
+interface RegionConfig {
+  geojsonFile: string;
+  updateS3Dir: string;
+  processGeojsonFlags?: string[];
+}
+
+interface ProcessingConfig {
+  // Country code
+  [key: string]: {
+    // Region code
+    [key: string]: RegionConfig;
+  };
+}
+
+export default class BulkReprocessRegions extends Command {
+  static description = `use a configuration file to process and update many regions`;
+
+  static flags = {
+    dryRun: flags.boolean({
+      allowNo: false,
+      description: `Dry run; only prints actions that would be taken.`
+    })
+  };
+  static args: [IArg] = [
+    {
+      name: "configFile",
+      required: true,
+      description: `Path to a configuration file containing information on how each region should be processed.
+
+The configuration file should be a JSON file with the following format:
+{
+  "US": {
+    "DE": {
+      "geojsonFile": "data/input/de.geojson",
+      "updateS3Dir": "s3://path/to/timestamped/data/files/like/US/DE/2021-09-23T18:43:42.300Z/",
+      "processGeojsonFlags": [
+        "-n",
+        "12,4,4",
+        "-x",
+        "12,12,12",
+        "-d",
+        "population:population,white:white,black:black,asian:asian,hispanic:hispanic,native:nativeAmerican,pacific:pacificIslander"
```

Comment:
No need to provide key-value pairs if you're not renaming them:
```suggestion
        "population,white,black,asian,hispanic,native:nativeAmerican,pacific:pacificIslander"
```

Reply:
👍 I just pulled this from Slack somewhere, good to know!